keymage.js

```diff
@@ -8,8 +8,7 @@
 (function(define, undefined) {
 define(function() {
     var VERSION = '1.1.3';
-    var isOsx = typeof navigator !== 'undefined' &&
-        ~navigator.userAgent.indexOf('Mac OS X');
+    var isOsx = /^mac/i.test(navigator.platform);
```

Comment:
https://developer.mozilla.org/ru/docs/Web/API/NavigatorID/platform - it's deprecated. Plus using regexp is going to be slower than indexOf anyway.